src/main/java/org/pac4j/demo/dw/Pac4JDemoApplication.java

```diff
@@ -0,0 +1,52 @@
+package org.pac4j.demo.dw;
+
+import org.eclipse.jetty.server.session.SessionHandler;
+import org.glassfish.hk2.utilities.binding.AbstractBinder;
+import org.pac4j.core.config.Config;
+import org.pac4j.demo.dw.resources.ViewsResource;
+import org.pac4j.dropwizard.Pac4jBundle;
+import org.pac4j.dropwizard.Pac4jFactory;
+
+import io.dropwizard.Application;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.setup.Environment;
+import io.dropwizard.views.ViewBundle;
+
+public class Pac4JDemoApplication extends Application<Pac4JDemoConfiguration> {
+
+    public static void main(String[] args) throws Exception {
+        new Pac4JDemoApplication().run(args);
+    }
+
+    private final Pac4jBundle<Pac4JDemoConfiguration> pac4j = new Pac4jBundle<Pac4JDemoConfiguration>() {
```

Comment:
I don't understand what you mean :)

This code is specific to the demo application, each application must have its own anonymous subclassing of `Pac4jBundle` with respect to their own subclassing of `Configuration`.